rls-analysis/src/analysis.rs

```diff
@@ -103,6 +103,34 @@ pub struct Def {
     // pub sig: Option<Signature>,
 }
 
+pub type Idents = HashMap<PathBuf, IdentsByLine>;
+pub type IdentsByLine = BTreeMap<Row<ZeroIndexed>, IdentsByColumn>;
+pub type IdentsByColumn = BTreeMap<Column<ZeroIndexed>, IdentBound>;
+
+#[derive(new, Clone, Debug)]
+pub struct IdentBound {
```

Comment:
Could you please add a comment what is this used for? I'm having a hard time understanding it from the definition here